lib/dialects/oracledb/index.js

```diff
@@ -0,0 +1,167 @@
+
+// Oracledb Client
+// -------
+'use strict';
+
+var _ = require('lodash');
+var inherits = require('inherits');
+var Client_Oracle = require('../oracle');
+var ColumnCompiler = require('./schema/columncompiler');
+var ReturningHelper = require('../oracle/utils').ReturningHelper;
+var Promise = require('../../promise');
+var stream = require('stream');
+var async = require('async');
```

Comment:
`async` is currently in `devDependencies` in the package.json. Since it's being used in the dialect code it should be moved to `dependencies` right?